tutorials/nextjs-app-serverless-functions-sqldb/index.mdx

```diff
@@ -0,0 +1,948 @@
+---
+meta:
+  title: Deploy a Next.js application using Serverless Containers and Serverless SQL Databases
+  description: This page explains how to deploy your fully serverless Next.js application using Serverless Containers and Serverless SQL Databases
+content:
+  h1: Deploy a Next.js application using Serverless Containers and Serverless SQL Databases
+  paragraph: This page explains how to deploy your fully serverless Next.js application using Serverless Containers and Serverless SQL Database
+dates:
+  validation: 2024-02-29
+  posted: 2024-02-29
+---
+
+This tutorial will guide you through the deployment of a fully serverless Next.js application using a Serverless Container and a Serverless SQL Database in a few minutes.
+
+You can either deploy your application:
+  - [step by step using Scaleway CLI](#deploy-next.js-application-frontend-using-serverless-containers) to understand each detailed action performed, and the resources and subresources required
+  - [using a Terraform template](#deploy-next.js-application-using-terraform-templates) to deploy your application faster and have a ready-to-use Infrastructure as Code.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)
+  - You have installed and initialized the [Scaleway CLI](https://www.scaleway.com/en/docs/developer-tools/scaleway-cli/quickstart/)
+  - You have installed [Docker Engine](https://docs.docker.com/engine/install/)
+  - You have installed [Node 18 or newer](https://nodejs.org/en/download)
+</Message>
```

Comment:
I think you should add the having created an API key requirement, in the requirements section

Reply:
@nerda-codes added it, since I was already reviewing it :)